src/classes/pinger.ts

```diff
@@ -1,5 +1,5 @@
 export default class Pinger {
-	#pinger?: ReturnType<typeof setTimeout>;
+	#pinger?: NodeJS.Timer | number;
```

Comment:
we should be environment generic as we support not only NodeJS but also Deno and so on...

Reply:
Yes should be ReturnType<...>